### DISCUSSION TITLE
Hello, regarding the react transactions coding test

### DISCUSSION THREAD

**Discussion by DavidNjoroge** [DevSkillsHQ/discusssions#71] [Help] [answered]
Hello, regarding the react transactions coding test, am using a backend api for crud operations but it does not seem to persist new accounts and handle the balance logic, how do I handle that, 
1 do I leave it like that assuming in real world scenario the backend would work
2 do I mock the backend logic on the cypress part
3 handle the amounts calculations on the react side
4 any other solution?

**Comment by fenske** [comment, marked as answer]
Hi @DavidNjoroge. I'm Anton from Alva. If I understand correctly, you're working on a full-stack coding task where the idea is to build the backend by yourself, and in that case, you need to ensure there is a persistence layer (an in-memory data store in this case).

The accounting API spec is given to you as a reference to know what endpoints to implement. 

Hope this makes sense. Let me know if I can clarify further.

BR,
Anton.
